.github/workflows/clang_tidy.yml

```diff
@@ -144,9 +148,64 @@ jobs:
       - name: Show sccache stats
         run: ${SCCACHE_PATH} --show-stats
 
+  Build-riscv64:
+    needs: [Docker, Smart_CI]
+    if: "!needs.smart_ci.outputs.skip_workflow"
+    timeout-minutes: 150
+    defaults:
+      run:
+        shell: bash
+    runs-on: aks-linux-16-cores-32gb
+    container:
+      image: ${{ fromJSON(needs.docker.outputs.images).ov_build.ubuntu_22_04_riscv_xuantie }}
+      volumes:
+        - /mount:/mount
+      options: -e SCCACHE_AZURE_BLOB_CONTAINER -e SCCACHE_AZURE_CONNECTION_STRING
```

Comment:
If you would like to use the remote `sccache` cache, you need to add a unique key in the env like https://github.com/openvinotoolkit/openvino/blob/3f99c9c1c801256c9ab6f2e85f49a9712ca2ec85/.github/workflows/clang_tidy.yml#L107

and use `sccache` as the launcher like https://github.com/openvinotoolkit/openvino/blob/3f99c9c1c801256c9ab6f2e85f49a9712ca2ec85/.github/workflows/clang_tidy.yml#L97

If you do not want to use cache, remove the `options` key, but it is recommended to use cache to speed up the pipelines.

Reply:
Thank you for the suggestion, done!